README.md

```
@@ -195,7 +195,20 @@ npm run lint     # Run ESLint
+
+## 👥 Contributors
+
+We appreciate all contributors who help improve DraftDeckAI ❤️
+
+### 📊 All Contributors
+https://github.com/Muneerali199/Draftdeckai/graphs/contributors
+
+### 🎨 Visual Contributors
 
+<a href="https://github.com/Muneerali199/Draftdeckai/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=Muneerali199/Draftdeckai" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)